.github/workflows/docker.yml

```diff
@@ -65,7 +65,11 @@ jobs:
           repository: csesoc/deployment
           token: ${{ secrets.GH_TOKEN }}
       - name: Install yq - portable yaml processor
+<<<<<<< HEAD
```

Comment:
may have been missed by ur editor but these conflicts should be resolved

Reply:
If you're having trouble with merging the json and binary files, rather than deal with each conflict manually, you can choose to keep the whole file as the incoming or current.
Looks like
```
git merge <incoming_branch_name>
git checkout --ours <files you want to keep your current branch's version of>
git checkout --theirs <files you want to keep the incoming version of>
```
Especially useful for conflicts in the huge json files because typically, you want to just pull those in and then perhaps rerun the processors.